BusinessTampereTrafficMonitoring/traffic_lights/api_client.py

```diff
@@ -29,6 +28,15 @@ class TrafficLightCycle(Base):
 
 class TrafficLightAPIClient:
     def __init__(self, url: str, monitored_devices: List[str], db: str):
+        """
+        Constructor for initializing TrafficLightAPIClient class
+
+        #Parameter:
+        # argument1: url (String)
+        # argument2: list of monitored devices List[string]
+        # argument3: database connection string (String)
```

Comment:
The format of the database connection string should be documented, or at least mention it's a database URL, as supported by SQL Alchemy. You can read more about that in SQL Alchemy's documentation: https://docs.sqlalchemy.org/en/14/core/engines.html#database-urls